src/components/combo_box/_combo_box.scss

```diff
@@ -76,6 +76,20 @@
     &.ouiComboBox__inputWrap-isLoading.ouiComboBox__inputWrap-isClearable {
       @include ouiFormControlLayoutPadding(3); /* 2 */
     }
+
+    &--inGroup#{&}--compressed {
+      height: $ouiFormControlLayoutGroupInputCompressedHeight; /* 2 */
+      line-height: $ouiFormControlLayoutGroupInputCompressedHeight; /* 2 */
+
+      .ouiComboBoxPill,
+      .ouiComboBoxPill + .ouiComboBoxPill {
+        margin: $ouiSizeXS $ouiSizeXS 0 0;
+      }
+
+      .ouiComboBoxPill--plainText {
+        line-height: calc($ouiSizeL - 2px);
```

Comment:
Why ouiSizeL? What is the 2px based on? borders?

Reply:
`$ouiSizeL` is the original value for the `line-height` of `.ouiComboBoxPill--plainText`. However, all in-group elements have a `- 2px` either by using `$ouiFormControlLayoutGroupInputCompressedHeight` or when the values are not standard, by directly performing the `- 2px`. This is because the group wrapper adds a top and bottom border around the entire group.